openpype/hosts/fusion/api/lib.py

```diff
@@ -18,12 +18,14 @@
     legacy_io,
 )
 from .pipeline import get_current_comp, comp_lock_and_undo_chunk
-
+from openpype.api import (
```

Comment:
This should be changed to `from openpype.pipeline.context_tools import get_current_project_asset`